mobile/src/bundles/users/components/image-picker/image-picker.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { PermissionsAndroid } from 'react-native';
+import { type StyleProp, type ViewStyle } from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import { type ImagePickerResponse } from 'react-native-image-picker';
+
+import { Button } from '~/bundles/common/components/components';
+import { ButtonType } from '~/bundles/common/enums/enums';
+
+type ImagePickerProperties = {
+    label: string;
+    containerStyle?: StyleProp<ViewStyle>;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const getImageFromLibrary = async (): Promise<ImagePickerResponse> => {
+    return await launchImageLibrary({ mediaType: 'photo' });
```

Comment:
I think you don't need await in this case, unless it is a specific case
(http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea)

```suggestion
    return launchImageLibrary({ mediaType: 'photo' });
```